mods/cnc/chrome/settings.yaml

```diff
@@ -543,4 +550,3 @@ Container@SETTINGS_PANEL:
 			Width: 140
 			Height: 35
 			Text: Reset
-
```

Comment:
Actually there were two newlines at end of file, my editor automatically removes trailing newlines and keeps only one (on-save). You can tell because github otherwise marks it in a red "return-style" arrow if the last newline is removed. 
I don't mean to nitpick, but it's actually really annoying (and prone to be forgotten) to have to open up nano/different editor and manually add however many extra trailing newlines were in the file pre-save (and have to do it again whenever I amend something :()
So I'm hoping this is ok, since I don't think trailing newlines are desirable anyway, and they were probably there by accident.